### DISCUSSION TITLE
协议分析的非常棒

### DISCUSSION THREAD

**Discussion by DesolateVE** [Geocld/XStreaming#48] [General]
我最近也在研究 greenlight 是如何串流 xbox 的，但既没学过 ts 也没接触过 electron 框架，整个代码调试加分析起来非常吃力，看了好兄弟发的文档，思路瞬间清晰多了，感谢分享

**Comment by DesolateVE** [comment]
看了你的共建计划，我是准备写 pc 端的，不过是用 c++ 写，可惜了

**Reply by kevinmore** [reply]
@DesolateVE 请问你最后写了c++版本吗？

**Reply by DesolateVE** [reply]
@kevinmore 你那边有进展了记得@我，我学习一下

**Reply by kevinmore** [reply]
> @kevinmore 你那边有进展了记得@我，我学习一下

我打算写个native的swift app，也和greenlight的作者取得了联系。奈何他和本项目作者都没有做C/C++版本的打算。而且本项目最新的iOS branch用的是objective-C + ReactNative，对我的项目几乎没有用。所以我打算直接用swift重写了。

**Comment by TpaeC** [comment]
请问你c++版本写完了吗

**Reply by TpaeC** [reply]
@DesolateVE  请问你c++版本写完了吗

**Reply by DesolateVE** [reply]
差不多是写完了，怎么了

**Reply by TpaeC** [reply]
请问怎么联系你，企鹅或者Vx
